src/pages/List/List.js

```diff
@@ -1,9 +1,87 @@
-import React from 'react';
-import { withRouter } from 'react-router-dom';
+import React, { useState, useEffect } from 'react';
 import styled from 'styled-components';
+import axios from 'axios';
+import Result from './Result';
+import Filters from './Filters';
+import ListContents from './ListContents';
 
 function List() {
-  return;
+  const [data, setData] = useState({ productList: [], items: 10, preItems: 0 });
+
+  const getData = () => {
+    axios
+      .get('/data/List.json')
+      .then(res => {
+        let result = res.data.slice(data.preItems, data.items);
+        setData(prev => {
+          return { ...prev, productList: [...data.productList, ...result] };
```

Comment:
- 여기서는 data.productList 가 아니라, prev.productList 가 되어야 할 것 같습니다!
```suggestion
          return { ...prev, productList: [...prev.productList, ...result] };
```